src/components/news/single-news-view/NewsContent.js

```
@@ -0,0 +1,57 @@
+        author={news.author?.name || 'Unknown Author'} 
+        date={formatDate(news.creationDate)}
+        tags={news.tags || []}
+      />
+
+      <NewsImage 
+        imagePath={news.imagePath} 
+        defaultImage={defaultImage} 
+        title={news.title} 
+      />
+
+      <div className="news-text" dangerouslySetInnerHTML={{ __html: news.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)